accelerator/core/apis.h

```diff
@@ -200,7 +200,7 @@ status_t api_send_trytes(const ta_config_t* const info, const iota_config_t* con
 status_t api_get_node_status(const iota_client_service_t* const service, char** json_result);
 
 /**
- * @brief Fetch transaction information with UUID.
+ * @brief Fetch buffered requests information with UUID.
```

Comment:
Shuold it be requests or request? The function name and the description of json_result are singular.